jekyll/_cci2/caching-strategy.md

```diff
@@ -77,6 +77,50 @@ Check #partial-dependency-caching-strategies to see if there are tips for the la
 
 If you find that a cache is not being restored, see [this support article](https://support.circleci.com/hc/en-us/articles/360004632473-No-Cache-Found-and-Skipping-Cache-Generation) for tips.
 
+### Cache unused or superfluous dependencies
+{: #cache-unused-or-superfluous-dependencies }
+
+Depending on what language and package management system you are using, you may be able to leverage tools that clear or “prune” unnecessary dependencies.
+
+For example, the node-prune package removes unnecessary files (markdown, typescript files, etc.) from `node_modules`.
+
+### Check if jobs need pruning
+{: #check-if-jobs-need-pruning}
+
+If you notice your cache usage is high and would like to reduce it:
+
+* Search for the `save_cache` and `restore_cache` commands in your `.circleci/config.yml` file to find all jobs utilizing caching and determine if their cache(s) need pruning.
+* Narrow the scope of a cache from a large directory to a smaller subset of specific files.
+* Ensure that your cache `key` is following [best practices]({{ site.baseurl}}/2.0/caching/#further-notes-on-using-keys-and-templates).
+
+{% raw %}
+```sh
+     - save_cache:
+         key: brew-{{epoch}}
+         paths:
+           - /Users/distiller/Library/Caches/Homebrew
+           - /usr/local/Homebrew
+```
+{% endraw %}
+
+Notice in the above example that best practices are not being followed. `brew-{{ epoch }}` will change every build causing an upload every time even if the value has not changed. This will eventually cost you money, and never save you any time. Instead pick a cache `key` like the following:
+
+{% raw %}
+```sh
+     - save_cache:
+         key: brew-{{checksum “Brewfile”}}
+         paths:
+           - /Users/distiller/Library/Caches/Homebrew
+           - /usr/local/Homebrew
+```
+{% endraw %}
+
+This will only change if the list of requested dependencies has changed. If you find that this is not uploading a new cache often enough, include the version numbers in your dependencies.
+
+Let your cache be slightly out of date. In contrast to the suggestion above where we ensured that a new cache would be uploaded any time a new dependency was added to your lockfile or version of the dependency changed, use something that tracks it less precisely.
```

Comment:
Is this paragraph part of the same bullet point (best practices for cache `key`)? I wasn't really able to tell when I should consider this option, and not track the dependency more precisely.

Reply:
Hm, I honestly didn't review the caching pages as they have their own tickets associated with them coming up. I just added the content that needed to be here rather than in the overview. I'll take a look thanks!

Reply:
@kbatuigas I had to go back and read the whole thing in context because it does seem confusing here. So yes, ALL the code and the last bit of information are a part of the "best practices" bullet point. I do agree it seems separated. I wonder if I can somehow make it indented like how it happens for numbered lists.

Reply:
Okay I think it worked! It's the commit below: [Add indent for bullet list](https://github.com/circleci/circleci-docs/pull/6369/commits/20e07df0073f3adcbf5a1213c06805edb8ef25ff) 👍